### PR TITLE
Update yeoman generators

### PR DESCRIPTION
**Pull request by mbwhite**
Specifically for the Angular Generator

- replace the network name, enroll id enroll secret etc questions with the name of card
- updated the package.json of the resultant app to use the new syntax for the rest server,.

### CODE REVIEW

File: packages/composer-client/lib/businessnetworkconnection.js [hunk 1]
```diff
@@ -455,7 +455,15 @@ class BusinessNetworkConnection extends EventEmitter {
             })
             .then((connection) => {
                 LOG.exit(method);
-                return this._connectionLogin(connection,this.card.getUserName(),this.card.getEnrollmentCredentials().secret);
+
+                let secret = this.card.getEnrollmentCredentials();
+                if (!secret){
+                    secret='na';
```
Comment (caroline-church) [inline packages/composer-client/lib/businessnetworkconnection.js]: can't this just be null

Comment (caroline-church) [reply packages/composer-client/lib/businessnetworkconnection.js]: apparently not because it has to be a string

**Comment by sstone1** [comment]
@mbwhite your unit tests don't work! :rage:

**Comment by sstone1** [comment]
Tests fixed in PR #2639.

**Pull request closed without merging**